byzcoin/service_test.go

```diff
@@ -394,6 +394,69 @@ func TestService_AddTransaction_ValidInvalid(t *testing.T) {
 	require.Nil(t, err)
 }
 
+// Sends the same transaction to two different nodes and makes sure that it shows up only once in a
+// block.
+func TestService_AddTransaction_Parallel(t *testing.T) {
+	defer log.SetShowTime(log.ShowTime())
+	log.SetShowTime(true)
+	s := newSerN(t, 1, testInterval, 4, disableViewChange)
+	defer s.local.CloseAll()
+
+	// add the first tx to create the instance
+	log.Lvl1("Adding a tx twice")
+	dcID := random.Bits(256, false, random.New())
+	tx1, err := createOneClientTxWithCounter(s.darc.GetBaseID(), dummyContract, dcID, s.signer, 1)
+	require.Nil(t, err)
```

Comment:
Could you change those occurrences to be `require.NoError(t, err)` ?

Reply:
Here and forever after...